tests/unit/go/task.yaml

```diff
@@ -0,0 +1,31 @@
+summary: Run project static and unit tests
+
+systems: [ubuntu-16.04-64]
+
+restore: |
+    rm -rf /tmp/static-unit-tests
+
+execute: |
+    mkdir -p /tmp/static-unit-tests/src/github.com/snapcore
+    cp -ar $PROJECT_PATH /tmp/static-unit-tests/src/github.com/snapcore
+    chown -R test:12345 /tmp/static-unit-tests
+
+    # remove leftovers
+    rm -r /tmp/static-unit-tests/src/github.com/snapcore/snapd/vendor/*/
+    rm -rf /tmp/static-unit-tests/src/github.com/snapcore/snapd/cmd/{autom4te.cache,configure,test-driver,config.status,config.guess,config.sub,config.h.in,compile,install-sh,depcomp,build,missing,aclocal.m4,Makefile,Makefile.in}
```

Comment:
We've just created this directory, right? Why do we need to remove specific files as is being done here? How about having this instead:
```
rm -rf /tmp/static-unit-tests
mkdir -p /tmp/static-unit-tests/src/github.com/snapcore
```
?